src/app/editor-common/app-editor-base/app-editor-base.ts

```diff
@@ -914,4 +915,24 @@ export abstract class AppEditorBase extends DirectiveBase implements StatusContr
             comp.job    = job;
         });
     }
+
+    exportApp() {
+        const serializedModel = this.dataModel.serialize();
```

Comment:
use `this.dataModel.serializeEmbedded` for workflows, then you won't have to resolve everything later.